### DISCUSSION TITLE
A603 Carrier Board | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#1575] [Q&A]
# A603 Carrier Board | Seeed Studio Wiki

A603 Carrier Board

https://wiki.seeedstudio.com/reComputer_A603_Flash_System/

**Comment by MSZ98** [comment]
I can't get step 6 (flash) to work. I have A603 Carrier Board, Orin Nano (P3767) module. Reading board information failed. The device is visible in recovery mode. Tried with JP 5.1.1 and JP 5.1.2. I'm using SD Card only. The only way I did it was the standalone image creation and flash SD with Etcher.

sudo ./tools/jetson-disk-image-creator.sh -o sd-blob.img -b jetson-orin-nano-devkit -d SD

**Comment by hmahadik** [comment]
I wasn't able to flash an Orin Nano (P3767) module either. Specifically, flashing an external 1 TB NVMe (2230) connected to the A603 carrier board was failing using the above guide for R35.4.1/JP5.1.2.

What worked for me was to use the command documented in the [official Nvidia documentation](https://docs.nvidia.com/jetson/archives/r35.4.1/DeveloperGuide/text/SD/FlashingSupport.html#using-initrd-flash-with-orin-nx-and-nano) (instead of Step 6 documented in this guide): 

```
sudo ./tools/kernel_flash/l4t_initrd_flash.sh --external-device nvme0n1p1 -p "-c ./bootloader/t186ref/cfg/flash_t234_qspi.xml" -c ./tools/kernel_flash/flash_l4t_t234_nvme.xml --showlogs --network usb0 jetson-orin-nano-devkit nvme0n1p1
```

**Comment by Alexanderbayer** [comment]
Your step 1 for JP6.1 is not correct, it downloads the wrong versions, it should download 36.4 in both commands. It worked when I manually downloaded the files from the Nvidia website here: https://developer.nvidia.com/embedded/jetson-linux-r3640

**Reply by nfs0619** [reply]
Thank you for your feedback and for pointing that out! The download link in the previous steps was indeed incorrect. We have now updated the link, and it will correctly download version 36.4. Thanks again for your help!

**Comment by neoscell** [comment]
After following this for JP6.1 with an Orin Nano 8 Gb my USB ports do not work??? 
It seems to have flashed otherwise… ?

**Reply by neoscell** [reply]
Seed Studio - Check your Forum and Discord Pages - people are constantly having issues with the USB not working and there is no clear solution. Please help!

**Reply by yuyoujiang** [reply]
I tried flashing JP6.1 onto the A603 carrier board with the Orin Nano 8GB module. After successfully executing the flashing command in step 4 (`sudo ./tools/kernel_flash/l4t_initrd_flash.sh --external-device nvme0n1p1 -c tools/kernel_flash/flash_l4t_external.xml -p "-c bootloader/t186ref/cfg/flash_t234_qspi.xml" --showlogs --network usb0 jetson-orin-nano-devkit internal`), the device functioned normally, including the USB ports.

**Comment by raj-anadkat** [comment]
The USB ports dont seem to be working with Jepack 6.1 on Orin Nano 8GB. Please help.

**Reply by nfs0619** [reply]
We have carefully reviewed the issue you mentioned, but our developers have not encountered the same problem during testing. 

![A603_Running_Pic](https://github.com/user-attachments/assets/e5b88a96-6784-465a-a18f-99bafd3d497c)
![A603_Connections](https://github.com/user-attachments/assets/e067a83f-c3c5-4da2-8292-b188f5bf7775)

To help ensure everything is set up correctly, we suggest following our official procedure to reflash the system.

If the issue persists, please provide more detailed information, such as:

- **Power specifications and connection method**  
- **Whether an external SSD is connected and its model**  
- **Specific operation logs or error messages**  

This information will help us identify and resolve the issue more efficiently. Thank you for your understanding and support!

**Comment by bw-turner** [comment]
I am encountering a few issues:
1) If I attempt to use the `l4t_create_default_user.sh` script and use a username that is anything other than `nvidia` the device is flashed successfully, however fails to boot.
2) Once flashed and connected as `nvidia` if I run `sudo apt update && sudo apt upgrade` the device fails to boot on a reboot.

I am able to work around the first of these issues, however it is quite important to be able to ensure that all the packages on the device are up to date. Please advise how to upgrade the packages on the device without rendering it unable to boot and requiring a re-flash.

**Reply by nfs0619** [reply]
Could you please provide screenshots of the error messages you encounter during the boot failure? This will help us better understand the issue and provide more accurate assistance. Thank you!

**Comment by stephendade** [comment]
Like others, I too had issues with the USB ports working and not being able use ``l4t_create_default_user.sh``.

I think I've solved it via:
1. Use ``sudo tools/l4t_create_default_user.sh -u {USERNAME} -p {PASSWORD} -a -n {HOSTNAME} --accept-license`` to create a user
2. For flashing use ``sudo ./tools/kernel_flash/l4t_initrd_flash.sh --external-device nvme0n1p1 -c tools/kernel_flash/flash_l4t_external.xml -p "-c bootloader/generic/cfg/flash_t234_qspi.xml" --showlogs --network usb0 jetson-orin-nano-devkit internal``. Note this command is slightly different to the Seeeed-provided one.

After this, I am able to login via ssh (headless) and the USB ports do work.

**Reply by stephendade** [reply]
@neoscell and @bw-turner, see if that works for you

**Reply by bw-turner** [reply]
Are you able to run `apt update && apt upgrade` successfully with this? And what platform are you using? I'm on an Orin Nano and wondering if that has something to do with it. I'm pretty sure I've done this before with an Orin NX without issue...

**Reply by stephendade** [reply]
Huh ... ``sudo apt update && apt upgrade`` broke USB support :cry:   It upgraded the Jetson from Jetpack 6.1 (L4T 36.4.0) to Jetpack 6.2 (L4T 35.4.3). So the provided Seed BSP files aren't compatible with 6.2

**Reply by neoscell** [reply]
@stephendade This did not solve my problem with the USB ports. Still not able to get a mouse moving :-(  I appreciate the help! I think my next move is to try installing 6.0 and just keep working backwards until I find something that works... or assume my board is broken??? 

From what I understand, there is risk with running apt update && apt upgrade to a Jetpack Linux build in that it will break/overwrite the Jetpack customization. That being said, I have accidentally installed updates while shutting down and it still seemed to work OK with my developer kit breakout board. I am surprised that upgraded the Jetpack version. I thought the NVIDIA SDK / flashing like this was the only way to do that... so shows how little I know :-P

**Reply by stephendade** [reply]
@neoscell, I did discover that if I simply re-ran the flashing command (``l4t_initrd_flash.sh ``) a second time, the board wouldn't boot for some reason. I had to delete the entire ``Linux_for_Tegra`` folder and start from the beginning. Maybe that helps?

Yes, very surprised that NVIDIA allows an auto-upgrade to JP6.2. I'm wondering it that's accidental...

**Reply by neoscell** [reply]
I don't have an issue with the board booting. It loads Linux up and logs in. Looks like no power to the USB.

I checked the Linux_for_Tegra folders and the files are copying into that directory. I don't know where I am going wrong. Every time I get a successful flash and boot-up.

**Comment by matt-nolan11** [comment]
Like many others here, I'm having issues with the USB-A ports. I'm using an Orin Nano 8GB (from the developer kit, if that matters) with the A603 carrier and a [512GB SK Hynix SSD](https://a.co/d/5g8sr60). The board boots fine, HDMI works, and I can ssh into it if I set up the user / hostname before flashing, but the USB-A ports aren't delivering power. If I run lsusb, I can see the USB hubs attached, so I'm guessing the data lines are fine, but the ports are definitely not delivering power.

The USB ports appear to properly deliver power at the start of the boot sequence, but at some point, they always shut off. This behavior is visible via the LEDs on my USB keyboard. If I leave the Force Recovery jumper installed, it takes a lot longer for the USB ports to stop delivering power.

There is an open issue on the Seeed Studio Wiki [Github](https://github.com/Seeed-Studio/wiki-documents/issues/2058), but there are no real updates yet.

I'm not that knowledgeable when it comes to these flashing details, but I banged some rocks together and ran a couple "experiments" (all using JP6.1):
* Flashed using Nvidia SDK Manager
  * Same exact behavior, the board boots and HDMI works, but no USB power
* Flashed using the above instructions for the A603, while the module was actually installed in the dev kit carrier board
  * No video from the DisplayPort, but the USB ports work fine

This definitely implies that either something is wrong with the provided drivers, or the copy-and-replace command is not working properly (not sure why this would be the case, I ran it with sudo per the instructions)

I'll probably have to work on my project using the dev kit carrier board until this issue is resolved.

**Reply by matt-nolan11** [reply]
I should note that I also had strangely inconsistent ssh connections over Ethernet, and I would get disconnected every few minutes. Might not be related, but could be worth mentioning in case others saw something similar

**Reply by nfs0619** [reply]
We have carefully reviewed the issue you mentioned, but our developers have not encountered the same problem during testing. 

![A603_Running_Pic](https://github.com/user-attachments/assets/e5b88a96-6784-465a-a18f-99bafd3d497c)
![A603_Connections](https://github.com/user-attachments/assets/e067a83f-c3c5-4da2-8292-b188f5bf7775)
<img width="1091" alt="A603_Flash_Log" src="https://github.com/user-attachments/assets/a95de73c-23ef-49c8-ae77-3d4cdcfa43c6" />

To help ensure everything is set up correctly, we suggest following our official procedure to reflash the system.

If the issue persists, please provide more detailed information, such as:

- **Power specifications and connection method**  
- **Specific operation logs or error messages**  

This information will help us identify and resolve the issue more efficiently. Thank you for your understanding and support!

**Reply by stephendade** [reply]
Hi @nfs0619:

For the USB issue, I am powering the Jetson with the supplied power connection and used the instructions for installing JP6.1

I had 1x camera connected via the USB3 ports (next to Ethernet jack):
```
Bus 002 Device 002: ID 04b4:0497 Cypress Semiconductor Corp. Arducam B0497 (USB3 8.3MP)
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 003: ID 8087:0a2b Intel Corp. Bluetooth wireless interface
Bus 001 Device 002: ID 05e3:0608 Genesys Logic, Inc. Hub
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
```

After doing ``sudo apt update && sudo apt upgrade && sudo reboot``, the USB camera was now missing:
```
Bus 002 Device 001: ID 1d6b:0003 Linux Foundation 3.0 root hub
Bus 001 Device 003: ID 8087:0a2b Intel Corp. Bluetooth wireless interface
Bus 001 Device 002: ID 05e3:0608 Genesys Logic, Inc. Hub
Bus 001 Device 001: ID 1d6b:0002 Linux Foundation 2.0 root hub
```

As others have mentioned, I checked the power on the USB3 ports and read 0V - so no power.

Running ``sudo dmesg | grep -i usb`` to look for issues:
```
[    0.080285] usbcore: registered new interface driver usbfs
[    0.080298] usbcore: registered new interface driver hub
[    0.080308] usbcore: registered new device driver usb
[    0.181990] ehci_hcd: USB 2.0 'Enhanced' Host Controller (EHCI) Driver
[    0.182215] ohci_hcd: USB 1.1 'Open' Host Controller (OHCI) Driver
[    0.182789] usbcore: registered new interface driver usb-storage
[    1.644653] usbcore: registered new interface driver usbhid
[    1.644658] usbhid: USB HID core driver
[    1.710657] usbcore: registered new interface driver snd-usb-audio
[    1.824843] tegra-xusb 3610000.usb: Adding to iommu group 2
[    1.840839] tegra-xusb 3610000.usb: Firmware timestamp: 2023-02-10 03:48:10 UTC
[    1.840846] tegra-xusb 3610000.usb: xHCI Host Controller
[    1.840860] tegra-xusb 3610000.usb: new USB bus registered, assigned bus number 1
[    1.841603] tegra-xusb 3610000.usb: hcc params 0x0180ff05 hci version 0x120 quirks 0x0000000000010810
[    1.841627] tegra-xusb 3610000.usb: irq 123, io mem 0x03610000
[    1.841735] tegra-xusb 3610000.usb: xHCI Host Controller
[    1.841738] tegra-xusb 3610000.usb: new USB bus registered, assigned bus number 2
[    1.841742] tegra-xusb 3610000.usb: Host supports USB 3.1 Enhanced SuperSpeed
[    1.842176] hub 1-0:1.0: USB hub found
[    1.842837] hub 2-0:1.0: USB hub found
[    2.178482] usb 1-3: new high-speed USB device number 2 using tegra-xusb
[    2.330968] hub 1-3:1.0: USB hub found
[    2.626483] usb 1-3.3: new full-speed USB device number 3 using tegra-xusb
[    5.300805] tegra-xudc 3550000.usb: Adding to iommu group 4
[    7.537299] fusb301 1-0025: failed to read device id, err : 0xffffff87
[    7.537304] fusb301 1-0025: fusb301 not support
[    7.537308] fusb301: probe of 1-0025 failed with error -22
[    7.895557] usbcore: registered new interface driver btusb
[   10.340669] usb0: HOST MAC 66:43:01:45:35:5c
[   10.340677] usb0: MAC 66:43:01:45:35:5d
```
Inserting an externally-powered USB device worked fine - device was detected as normal. So it does sound like some issue with the USB's power control.

EDIT: If it's relevant, I am running my Jetson headless via an SSH connection.

**Reply by nfs0619** [reply]
Hello,

Thank you for sharing the details of your issue! We have recently updated the **Jetson Usage** FAQ documentation to address issues like the one you're facing. You can refer to the following link for troubleshooting steps that might help resolve the USB power issue:

[FAQs for Jetson Usage - Seeed Wiki](https://wiki.seeedstudio.com/Jetson_FAQ/#q7-i-cannot-use-usb-a-port-ethernet-port-or-no-hdmi-dispaly-after-flashing-the-device)

If the issue persists after following the steps outlined in the FAQ, we recommend reaching out to our technical support team at **[techsupport@seeed.io](mailto:techsupport@seeed.io)**, and they will assist you further.

We appreciate your patience and understanding as we work to resolve this issue. Hopefully, this will help you continue with your project!

**Reply by n9rai** [reply]
there is no hint in the FAQ that solves the problem!

**Reply by johnmckenna-snd** [reply]
FWIW, on 6.2, I was able to get the USB drivers working and run `sudo apt update && sudo apt upgrade`. Stuff still seems to be working.

I did however make a super stupid mistake the first time so I opened a pr to help that. [PR Here](https://github.com/Seeed-Studio/wiki-documents/pull/2252)

**Comment by n9rai** [comment]
A fresh installation works as expected and everything (USB, HDMI) works. However, as soon as I install something, e.g. ROS2 and a suda apt update / upgrade is required in the installation process, no USB port works after a reboot and direct interaction with the Jetson Orin is impossible. Unfortunately, the carrier board is unusable in this state. If there is no way to install the ‘correct’ USB drivers via SSH after an update, I will unfortunately have to return the board.

**Reply by nfs0619** [reply]
Hello,

Thank you for sharing your issue, and we truly understand the frustration you're experiencing. According to the official ROS2 installation guide ([link](https://docs.ros.org/en/eloquent/Installation/Linux-Development-Setup.html)), the installation process doesn’t actually involve running `sudo apt upgrade`, and doing so may interfere with certain drivers, potentially causing the USB issue.

To avoid this, we recommend refraining from running `sudo apt upgrade` during the installation, and checking if any other drivers or software might be causing a conflict.

If the issue persists, please don’t hesitate to contact our technical support team at [techsupport@seeed.io](mailto:techsupport@seeed.io), and we’ll do our best to assist you in resolving the problem.

Thank you for your understanding and patience. We look forward to helping you resolve this issue!

**Reply by n9rai** [reply]
is there a way to reinstall the USB drivers on a running and updated system? Not to update is not a good solution. I have ssh and Remote Desktop running now and I am able to interact with the Jetson Orin Nano, but USB is not working. I also tried the newly added 6.2 installation but than my Wifi Card is not working.

**Reply by nfs0619** [reply]
Hello!

We have recorded a video tutorial to help you resolve the issue. You can view it here: [Video Link](https://www.youtube.com/watch?v=qN4GxscUGW0).

Additionally, we have updated the FAQ section on our Wiki. You can find answers to your issue in Q7 to Q9 here: [Jetson FAQ](https://wiki.seeedstudio.com/Jetson_FAQ/#q7-i-cannot-use-usb-a-port-ethernet-port-or-no-hdmi-dispaly-after-flashing-the-device).

If the problem persists, feel free to contact our tech support team at [techsupport@seeed.io](mailto:techsupport@seeed.io). We are happy to assist you further.

**Comment by neoscell** [comment]
Only the micro USB works for me. I have an adapter that goes to regular USB, so that’s my current workaround with 6.1. Finally got everything installed through SDK manager but had to go over Wi-Fi, even when I connect the micro USB to my host pc to install packages.

As far as I can tell, Seeed isn’t able to recreate the issue so they aren’t doing anything to fix it.

I haven’t tried 6.2, but was thinking of it until I saw this Wi-Fi issue… :-( @neurons9 which Wi-Fi adapter are using?

**Reply by n9rai** [reply]
@neoscell i have a Waveshare Wireless AX210 NIC. The Problem is the iwlwifi driver. Wifi was completely dead after fresh installation and I was not able to get it work

**Reply by neoscell** [reply]
I tried to update to 6.2 and then the USB WORKS! ... but the WiFi does NOT work!!! :-( AAAhhhhh
Wish this would just work! I am going to flash back to 6.1, so at least the WiFi works. I wonder if I do the upgrade to the system after that to get to 6.2 will it keep the WiFi or not? 
I am wasting so much time on this when I could be using the board in my application :-( 
We should be able to reliably upgrade this board to match JP upgrades, without worrying about these types of issues.

**Reply by matt-nolan11** [reply]
Same here, USB power is now working with JP6.2, and seems to persist after using sudo apt update / upgrade (though I assume the updating problem will come back as soon as a new JetPack version is released). However, WiFi is not working due to similar driver issues. 

Running lspci shows an identified "Realtek Semiconductor Co., Ltd. RTL8822CE 802.11ac PCIe Wireless Network Adapter" (I'm using the M.2 wireless adapter that came with the Orin Nano 8GB dev kit). So definitely a driver issue.

When I try to manually install drivers (with ChatGPT's help), I hit a wall due to a bunch of unknown symbols related to loading the cfg80211 module. Not sure if this info helps or is common to others

Trying to manually install drivers for a Realtek USB Wi-Fi adapter results in similar issues.

**Comment by nfs0619** [comment]
Hello everyone,

In JetPack 6.2, even after installing iwlwifi, the Intel AX200 wireless card may not work properly. This could be due to the rfkill module not being enabled in the new kernel, preventing the wireless card from functioning correctly.  

To resolve this issue, you can recompile the kernel and configure systemd accordingly. A detailed step-by-step guide is available here: https://docs.google.com/document/d/1tO0f4fSNvGluck_m6awRSI9I4uZIGVIshNnYmlrMK-s/edit?tab=t.0#heading=h.zd3ug3y2do1t .

Additionally, NVIDIA provides some Jetson L4T kernel customization methods, which might also be helpful: [NV Official Kernel Customization Guide](https://elinux.org/Jetson/L4T/r36.3.x_patches#Kernel_Customization)

If you're experiencing the same issue on the A603 carrier board, we recommend trying the above solutions. If you have any further questions, feel free to join the discussion on the [Seeed Studio Forum](https://forum.seeedstudio.com/) or reach out to us via email at [techsupport@seeed.io](mailto:techsupport@seeed.io). We're happy to help!

**Comment by eric-distributed-spectrum** [comment]
I installed Jetpack 6.2, now the hwclock command is not working properly.

sudo hwclock -f /dev/rtc0 --verbose
hwclock from util-linux 2.37.2
System Time: 1741967624.290900
Using the rtc interface to the clock.
Assuming hardware clock is kept in UTC time.
Waiting for clock tick...
hwclock: select() to /dev/rtc0 to wait for clock tick timed out
...synchronization failed

Anyone get the rtc to work on Jetpack 6.2?

**Reply by nfs0619** [reply]
Thank you for your feedback. We have received your message and are currently arranging for debugging. If you do not receive further updates, please feel free to reach out to our technical support at [techsupport@seeed.io](mailto:techsupport@seeed.io) for further assistance. Thank you for your support and understanding.

**Comment by robotjuhwan** [comment]
I heard that jetson orin super can be activated after Jetpack 6.2. It is possible to use orin nx super on this carrier board?

**Reply by nfs0619** [reply]
Yes, the Orin NX Super can be used on the A603 carrier board.

**Comment by MassarSara** [comment]
:

On JetPack 5.1.2 (L4T 35.4.1):
The flashing process consistently fails with Waiting for target to boot up..., despite proper recovery mode, USB connection, and clean NVMe preparation.

On JetPack 5.1.1 (L4T 35.3.1):
The flash tool returns a RuntimeError 3: USB delay during bootloader communication, preventing successful flashing.

On JetPack 5.1.4 (L4T 35.5.0):
The binarize.sh script fails to execute properly due to errors in handling the /dev folder inside the rootfs, which halts the flashing process.

These issues make it nearly impossible to use JetPack 5.x on the A603 board. I’ve tested all solutions carefully with no success.

**Reply by nfs0619** [reply]
Thank you for your feedback. We have received your message and are currently arranging for debugging. If you do not receive further updates, please feel free to reach out to our technical support at [techsupport@seeed.io](mailto:techsupport@seeed.io) for further assistance. Thank you for your support and understanding.

**Reply by MassarSara** [reply]
Thank you for your reply. I just wanted to follow up and kindly ask if there are any updates. I’m still waiting because I need to install the IMX477 driver on this board, and flashing JetPack 5.x is a prerequisite for that.

Looking forward to your guidance or any workaround in the meantime. Thanks again for your support!

**Comment by bribrah** [comment]
I am using a 256gb NVME SSD and that seems to be causing issues... 
```
***************************************
*                                     *
*  Step 3: Start the flashing process *
*                                     *
***************************************
Waiting for target to boot-up...
Waiting for target to boot-up...
Waiting for target to boot-up...
Waiting for target to boot-up...
Waiting for target to boot-up...
Waiting for target to boot-up...
Waiting for target to boot-up...
Waiting for target to boot-up...
Waiting for target to boot-up...
Waiting for target to boot-up...
Waiting for target to boot-up...
Waiting for target to boot-up...
Waiting for device to expose ssh ......Waiting for device to expose ssh ...Run command: flash on fc00:1:1:0::2
SSH ready
blockdev: cannot open /dev/mmcblk0boot0: No such file or directory
[ 0]: l4t_flash_from_kernel: Serial Number: 1611825602726
[ 0]: l4t_flash_from_kernel: Starting to create gpt for emmc
Active index file is /mnt/internal/flash.idx
Number of lines is 61
max_index=60
[ 2]: l4t_flash_from_kernel: Successfully create gpt for emmc
[ 2]: l4t_flash_from_kernel: Starting to create gpt for external device
Active index file is /mnt/external/flash.idx
Number of lines is 18
max_index=17
writing item=1, 9:0:primary_gpt, 512, 19968, gpt_primary_9_0.bin, 16896, fixed-<reserved>-0, 755b01fde4b56500a5888a7b31c8d6b8af71c9d3
Warning: Error fsyncing/closing /dev/nvme0n1: Input/output error
Error: end of file while reading /dev/nvme0n1
Error: No space left on device during write on /dev/nvme0n1
Warning: Error fsyncing/closing /dev/nvme0n1: Input/output error
Flash failure
Either the device cannot mount the NFS server on the host or a flash command has failed. Check your network setting (VPN, firewall,...) to make sure the device can mount NFS server. Debug log saved to /tmp/tmp.gVoDCwBcr1. You can access the target's terminal through "sshpass -p root ssh root@fc00:1:1:0::2" 
Cleaning up...
```
Not sure ow its running out of space? 256gb should be more than enough and is in fact a larger drive than my host system

**Reply by nfs0619** [reply]
Thanks for sharing the log snippet. To better assist you, I recommend sending the **full flashing log** along with your **hardware setup details** (e.g. SSD model, connection method, etc.) to our technical support team at **[techsupport@seeed.io](mailto:techsupport@seeed.io)**.

They'll be able to provide you with more detailed and faster troubleshooting support.

**Comment by agolgo** [comment]
I tried flashing a Jetson Orin Nano with JetPack 6.2, but on my first attempt, the USB didn’t work, even though I followed the steps exactly as shown here. So, I checked the driver files that were supposed to be replaced by the ones in the JetPack folder ("Linux_for_Tegra"). I found that for some reason, the command:

sudo cp -r 603_jp62/Linux_for_Tegra/* Linux_for_Tegra/

didn’t replace all the driver files. I ended up copying and replacing them manually in the JetPack folder, and after that, the USB worked fine!

**Comment by alirezaahmadi** [comment]
I accidentally burned the IC on my A603 board, which is a 6-pin chip exactly behind the main DC power jack. The code written on the IC is 4QMJ.
The order was not turning on until I took it from the board, and it could again turn on, which is why I suspect that it was an inverse power protection IC.  
know I am looking for the replacement chip to buy, but unfortunately, I am not able to find the exact part number of that chip. Does anyone know that is standard retail name-number of this chip is?

**Reply by Frank-Nie02** [reply]
Hi,

I recommend that you contact our technical support team at [techsupport@seeed.io](mailto:techsupport@seeed.io) and provide them with the related information. Our technical support team will help you further.
